kolibri/core/content/api.py

```diff
@@ -1310,6 +1315,79 @@ def consolidate(self, items, queryset):
         return sorted_items
 
 
+class ContentDownloadRequestViewset(ValuesViewset):
+    pagination_class = OptionalPageNumberPagination
+
+    queryset = ContentDownloadRequest.objects.all()
+
+    field_map = {
+        "source_id": "self.request.user.id",
```

Comment:
This won't work - the field_map maps from a field that is in the annotated queryset - and presumably we just want to get the `source_id` from the object itself? So can just add this to the `values` tuple.